pyproject.toml

```diff
@@ -0,0 +1,107 @@
+[tool.poetry]
+authors = ["Enercoop <team-data@enercoop.org>"]
+classifiers = [
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python :: 3",
+  "Operating System :: OS Independent",
+]
+description = "Tools to manipulate energy time-series and contracts, and to perform forecasts."
+exclude = ["tests"]
+license = "MIT"
+name = "enda"
+packages = [
+  {include = "enda"},
+]
+readme = "README.md"
+repository = "https://github.com/enercoop/enda"
+version = "0.1.0"
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/enercoop/enda/issues"
+
+[tool.poetry.dependencies]
+h2o = "^3.42.0.4"
+pandas = ">=1.0.0,<2.0.0"
+python = ">=3.7.3,<3.11"
+scikit-learn = "^1.0.2"
+statsmodels = "<=0.13.5"
```

Comment:
Would it be possible to make h2o and scikit-learn optional dependencies of the module ?

Reply:
I don't see how these could be optional. What models are available in Enda that do not require h2o or scikit-learn? In the example provided there's no way to use enda without them, see https://github.com/enercoop/enda/pull/11/files#diff-6b48093e73f3f978fb15a3c5a61a3e315bd5a4b6ff7ba4d8fa314c3a50cbe78bR162

Reply:
In fact, there might be two reasons to support them being optional dependencies: 
- enda offers some functions which are related to these ML backends, such as timeseries resampling for instance. One may imagine a use-case in which an end-user wants to use the resampling, but doesn't want to make a forecast. 
- an end-user might want to use only one of the two ML backends provided, eg. h2o but not scikit-learn. Here, it seems he will have to import both. 

To be honest, I am not sure it's a problem to have them as required dependencies as you propose. The main use of enda remains forecasting, and having these two packages imported is not a big deal.